list.js

```diff
@@ -0,0 +1,7 @@
+$("input[type='submit']").on("click",function(event){
+    event.preventDefault();
+    let txtBox=$("input[type='text']");
```

Comment:
Why use let instead of const here? Are you planning on re-assigning the variable later on? Would it more efficient to find and cache a reference to the input?